src/content/contact.js

```diff
@@ -0,0 +1,10 @@
+export default function generateContact() {
```

Comment:
- [ ] [REQUIRED] All `function` keywords like this one should be replaced with arrow functions.
![image](https://user-images.githubusercontent.com/73254040/99158361-877d8700-2697-11eb-8f85-bd02be22befa.png)
[Source](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/59622983/)